### PR TITLE
ci: Enable baremetal VFIO integration tests

### PR DESCRIPTION
**Pull request by sboeuf**
Relying on a NVIDIA Tesla T4 card present in the SGX machine, this patch
enables baremetal VFIO testing, validated by running several NVIDIA
tools in the guest. The guest image has been prepared to include all the
software needed to run these tests.

Signed-off-by: Sebastien Boeuf <sebastien.boeuf@intel.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -103,9 +103,8 @@ pipeline{
 						}
 					}
 				}
-				stage ('Worker build SGX') {
+				stage ('Worker build SGX & VFIO') {
 					agent { node { label 'bionic-sgx' } }
```
Comment (rbradford) [inline Jenkinsfile:107]: The better thing to do is add a new label `bionic-nvdidia-vfio` and assign it to the same node. And then add a new block that needs to run on that label.

File: tests/integration.rs [hunk 2]
```diff
@@ -6323,4 +6325,60 @@ mod tests {
...
+                    .ssh_command("sudo nv-hostengine")
+                    .unwrap()
+                    .trim()
+                    .contains("Started host engine"));
+                assert!(guest
+                    .ssh_command("sudo dcgmi discovery -l")
+                    .unwrap()
+                    .trim()
+                    .contains("Name: Tesla T4"));
+                assert!(
```
Comment (rbradford) [inline tests/integration.rs:6369]: This should be `assert_eq!`

File: Jenkinsfile (hunk 1 above)
Comment (sboeuf) [reply Jenkinsfile]: Done! I've renamed the node `bionic-sgx` into `bionic-baremetal`, and I've updated the list of labels from `bionic-sgx` to `bionic-sgx bionic-vfio`.

**Comment by sboeuf** [comment]
I'm fixing the image.

**Review by rbradford — APPROVED** [review]

**Pull request merged**